modules/Bio/EnsEMBL/Compara/PipeConfig/HomologyAnnotation_conf.pm

```diff
@@ -111,6 +113,15 @@ sub default_options {
         'old_server_uri'   => $self->o('compara_db'),
         'db_name'          => $self->o('dbowner') . '_' . $self->o('pipeline_name'),
 
+        # DB copy list of tables for per-species rapid release
+        'table_list' => [
+            "genome_db",
+            "method_link_species_set",
+            "gene_member",
+            "peptide_align_feature",
```

Comment:
This hasn't been discussed, but I figured since it carries the information for the bulk of the analyses, it is worth keeping/copying for now. It's possible it could be used later on to update/detect changes if a reference is updated. Again, not something discussed.